aca_preview/tests/test_checks.py

```diff
@@ -4,7 +4,7 @@
 from proseco import get_aca_catalog
 from proseco.core import StarsTable
 from proseco.tests.test_common import DARK40, STD_INFO, mod_std_info
-from aca_preview.preview import ACAReviewTable
+from ..preview import ACAReviewTable
```

Comment:
It looks like this was a case of me making two changes at once and deciding they were both fine instead of confirming they were both required.  This was how we had testing set in proseco, so I went with it when I saw something broken.  The something that was broken was really the lack of the `__init__py` file in the tests directory.  Without that `__init__.py` , py.test was, I think, loading both the wrong version of the tests and the wrong version of the code installed in my local conda env.

Reply:
Since it works relative or absolute and  @taldcroft mentions being on the fence, I'll plan to merge this as-is, but we can always swap it back in a tiny change.